src/test/java/org/minions/devfund/lourdes/ActorTest.java

```diff
@@ -0,0 +1,79 @@
+package org.minions.devfund.lourdes;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Implemented tests for {@link Actor}.
+ */
+public class ActorTest {
+    private static final String ACTOR_NAME = "Robert Downey";
+
+    /**
+     * Verify actor constructor with empty params.
+     */
+    @Test
+    public void testActorConstructor() {
+        Actor actor = new Actor();
+        assertEquals(actor.getName(), "");
```

Comment:
first argument is the expected result and the second one is the actual result.
Please update all your tests